src/pages/Layout.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import {Outlet} from "react-router-dom";
+import Navbar from "./Navbar";
+
+const Layout = () => {
+  return (
+    <>
+      <Navbar />
+      <Outlet />
+    </>
```

Comment:
Question(nb): Is the empty tag allowed in the return?

Reply:
yep! it's just part of the sytax for jsx